image.scala.yaml

```diff
@@ -4,6 +4,7 @@ name: "radanalytics-scala-spark"
 version: 1.0
 release: dev
 from: "radanalyticsio/openshift-spark:2.2-latest"
```

Comment:
shouldn't this change to be the new scala s2i base image?

Reply:
It should, but for some reason I rolled back the change because it was using my own image. Can we move tha base image to radanalyticsio namespace?